packages/core/http/core-http-server-internal/src/lifecycle_handlers.ts

```diff
@@ -86,7 +106,12 @@ export const registerCoreHandlers = (
   config: HttpConfig,
   env: Env
 ) => {
+  // add headers based on config
```

Comment:
We don't often update these and I added comments to more easily follow where they're applied.